web/themes/custom/novel/novel.theme

```diff
@@ -146,4 +147,55 @@ function novel_form_system_theme_settings_alter(array &$form, FormStateInterface
   // Remove the Path field from the logo and favicon upload fields.
   unset($form['logo']['settings']['logo_path']);
   unset($form['favicon']['settings']['favicon_path']);
+  // Add in identity settings.
+  $form['identity']['settings'] = [
+    '#type' => 'fieldset',
+    '#title' => t('identity settings'),
+  ];
+  $form['identity']['settings']['identity_color'] = [
+    '#type' => 'color',
+    '#title' => t('Identity Color', [], ['context' => 'Identity settings']),
+    '#default_value' => theme_get_setting('identity_color'),
+    '#description' => t('Choose library identity color.', [], ['context' => 'Identity settings']),
+  ];
+}
+
+/**
+ * Preprocesses variables for HTML templates.
+ */
+function novel_preprocess_html(array &$variables): void {
+  $identityColor = theme_get_setting('identity_color');
+
+  if ($identityColor) {
+    $hslColor = Hex::fromString($identityColor)->toHsl();
+    attach_identity_color_styles($variables, $hslColor);
+  }
+}
+
+/**
+ * Attaches identity color styles to the page.
+ *
+ * @param array<mixed> $variables
+ *   An associative array containing the following possible keys:
+ *   - '#attached': An array of attachments for rendering.
+ * @param \Spatie\Color\Hsl $hslColor
+ *   The HSL color object.
+ */
+function attach_identity_color_styles(array &$variables, Hsl $hslColor): void {
```

Comment:
```suggestion
function novel_attach_identity_color_styles(array &$variables, Hsl $hslColor): void {
```

In general it is a good practice to prefix functions with their module names in Drupal - even if it is not a hook implementation. This has a number of advantages:

- It signals what part of the project a particular function belongs to.
- We avoid collisions because functions in PHP files are not namespaced by default.